doc/paper/espnet-se++/paper.md

```diff
@@ -643,56 +227,9 @@ Calling `SeparateSpeech` and `Speech2Text` with unprocessed audios returns the s
 
 #### SSE 
 ![](https://i.imgur.com/skZ8uDP.png)
```

Comment:
Could you highlight the python code snippet before creating the screenshot? I think it will make the code more readable.